app/lib/Plugins/SearchEngine/Elastic8.php

```diff
@@ -273,10 +274,11 @@ private function addFragmentToUpdateContentBuffer(
 	protected function getClient(): Client {
 		if (!self::$client) {
 			$logger = new \Monolog\Logger('elasticsearch');
-			$log_level = Monolog\Logger::WARNING;
+			$log_level = self::LOG_LEVEL_DEFAULT;
 			if (defined('__CA_ELASTICSEARCH_LOG_LEVEL__')) {
 				$log_level = __CA_ELASTICSEARCH_LOG_LEVEL__;
 			}
+			$log_level = \Monolog\Logger::toMonologLevel($log_level);
```

Comment:
This was a little confusing at first, maybe we could use a different variable name for one of these or refactor this to make it more apparent?
(at first glance it seemed we were setting `$log_level` three times in a row with the final overriding the first two.)

Reply:
Will send this through as an update.